frontend/src/components/App/FishbowlList/index.tsx

```diff
@@ -19,7 +19,7 @@ import {
   ROUTE_FISHBOWL_HOST_NOW,
   ROUTE_HOME
 } from '@/app.config';
-import { Fishbowl } from '@/types/api-platform';
+import { Fishbowl } from '@/types/graphql-codegen/types';
```

Comment:
Here the fishbowls are get using axios, not graphql, so the type is not really correct....